integrationTest/integration_test.go

```diff
@@ -829,6 +838,27 @@ func makeTests(t *testing.T) []*TestGroup {
 				txtmulti("foo3", []string{strings.Repeat("X", 255), strings.Repeat("Y", 255), strings.Repeat("Z", 255)})),
 		),
 
+		testgroup("DS"
+			requires(providers.canUseDS),
+			tc("create a ds record", ds("@", 1, 13, 1, "ADIGEST")),
+			tc("create a ds record", ds("@", 65535, 13, 1, "ADIGEST")),
```

Comment:
Please give each "create a ds record" a unique string. It could be "ds variation A" through "ds variation Z" or numbers.